bHY6yh-d34855/cod-vyg67d_bHY6yh-d34855.xml

```diff
@@ -0,0 +1,85 @@
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="O" n="cod-vyg67d">Vatican, Biblioteca Apostolica Vaticana, Vat. lat. 8175</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-08-10" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+           <pb ed="#O" n="78-v"/>
+           <cb ed="#O" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d34855"><!-- p2l4-r12 -->
+            <head xml:id="bHY6yh-d34855-Hd1e108">Rubrica 12</head>
+            <head xml:id="bHY6yh-d34855-Hd1e111" type="question-title">De satisffactione illate iniurie</head>
+            <!--NOTE: the headings of rubric 12 and 13 appear to be mixed up, header of rubric 13 precedes heading of rubric 12 but the body paragraphs are in the correct order-->
+            <p xml:id="bHY6yh-d34855-d1e121">
+               <!--this paragraph precedes header, header has been moved to follow after this paragraph-->
+               ¶ <ref xml:id="bHY6yh-d34855-Rd1e124">ISidorus solilouiorum libro 2. capitulo 
+               <lb ed="#O"/>22.</ref> ¶ <quote xml:id="bHY6yh-d34855-Qd1e125">Si contristaueris in 
+               <lb ed="#O"/>aliquo fratrem tuum <sic>satiffac</sic> illi. Si pec<lb ed="#O" break="no"/>caueris 
+               in eum age penitentiam 
+               <lb ed="#O"/>coram illo. ¶ Si offenderis 
+               <lb ed="#O"/>cui quod repropitiare eum prece. Per<lb ed="#O" break="no"/>ge 
+               uelociter ad reconciliationem 
+               <lb ed="#O"/>proximi tui. ¶ Offensioni tue 
+               <lb ed="#O"/>cito ueniam postula. ¶ Non re<lb ed="#O" break="no"/>quiescas 
+               nisi reconciliatus fu<lb ed="#O" break="no"/>eris 
+               fratri.</quote>
+            </p>
+            <p xml:id="bHY6yh-d34855-d1e114">
+               ¶ De satiffatione illa<lb ed="#O" break="no"/>te 
+               iniurie. Rubrica. 12 
```

Comment:
Notice that there is a little "a" after the 12? and a "b" after the 13 in rubric. I think you can leave it in the position that you have it, but I think the 'a' and 'b' are the scribe's way of correcting this mistake.
```suggestion
               iniurie. Rubrica. 12 <sic>a</sic>
```